src/ExpressionEvaluator/CSharp/Test/ExpressionCompiler/PerfTests/DebuggerSteppingTestcase.xml

```diff
@@ -51,7 +49,6 @@
         <StepOver waitForBreakOrEnd="true"/>
       </Debug>
       <WaitForCodeMarkers />
-      <MeasureTimeStop /><!-- End timer (Not sure if correct command) -->
       <Debug>
         <StopDebugging />
```

Comment:
ya, you need to create one per each language and check right dlls are loaded per test. the code I pasted above is just an example, you should take a look at others and see how it is actually used.